packages/react/stories/testing.stories.tsx

```diff
@@ -167,868 +173,295 @@ export const Sizes: StoryFn = () => {
-      </List.Unordered>
-      <Spinner size='xs' title='Loading' />
-      <Spinner size='sm' title='Loading' />
-      <Spinner size='md' title='Loading' />
-      <Spinner size='xl' title='Loading' />
-      <div data-size='xs'>
-        <Modal.Context>
-          <Modal.Trigger>Open Modal</Modal.Trigger>
-          <Modal>
-            <Heading style={{ marginBottom: 'var(--ds-spacing-2)' }}>
-              Modal header
-            </Heading>
-            <Paragraph style={{ marginBottom: 'var(--ds-spacing-2)' }}>
-              Lorem ipsum dolor sit, amet consectetur adipisicing elit.
-              Blanditiis doloremque obcaecati assumenda odio ducimus sunt et.
-            </Paragraph>
-          </Modal>
-        </Modal.Context>
-      </div>
-      <div data-size='sm'>
-        <Modal.Context>
-          <Modal.Trigger>Open Modal</Modal.Trigger>
-          <Modal>
-            <Heading style={{ marginBottom: 'var(--ds-spacing-2)' }}>
-              Modal header
-            </Heading>
-            <Paragraph style={{ marginBottom: 'var(--ds-spacing-2)' }}>
-              Lorem ipsum dolor sit, amet consectetur adipisicing elit.
-              Blanditiis doloremque obcaecati assumenda odio ducimus sunt et.
-            </Paragraph>
-          </Modal>
-        </Modal.Context>
-      </div>
-      <div data-size='md'>
-        <Modal.Context>
-          <Modal.Trigger>Open Modal</Modal.Trigger>
-          <Modal>
-            <Heading style={{ marginBottom: 'var(--ds-spacing-2)' }}>
-              Modal header
-            </Heading>
-            <Paragraph style={{ marginBottom: 'var(--ds-spacing-2)' }}>
-              Lorem ipsum dolor sit, amet consectetur adipisicing elit.
-              Blanditiis doloremque obcaecati assumenda odio ducimus sunt et.
-            </Paragraph>
-          </Modal>
-        </Modal.Context>
-      </div>
-      <div data-size='xl'>
-        <Modal.Context>
-          <Modal.Trigger>Open Modal</Modal.Trigger>
-          <Modal>
-            <Heading style={{ marginBottom: 'var(--ds-spacing-2)' }}>
-              Modal header
-            </Heading>
-            <Paragraph style={{ marginBottom: 'var(--ds-spacing-2)' }}>
-              Lorem ipsum dolor sit, amet consectetur adipisicing elit.
-              Blanditiis doloremque obcaecati assumenda odio ducimus sunt et.
-            </Paragraph>
-          </Modal>
-        </Modal.Context>
-      </div>
-      <Pagination aria-label='Sidenavigering' data-size='xs'>
-        <Pagination.List>
-          <Pagination.Item>
-            <Pagination.Button aria-label='Forrige side' {...prevButtonProps}>
-              Forrige
-            </Pagination.Button>
-          </Pagination.Item>
-          {pages.map(({ page, itemKey, buttonProps }) => (
-            <Pagination.Item key={itemKey}>
-              <Pagination.Button {...buttonProps} aria-label={`Side ${page}`}>
-                {page}
+      {sizes.map((size) => (
```

Comment:
Cant you just use one `map` here and return components in a fragment or array? Thats quite a lot of maps...

Reply:
Thats what I originally did, but due to use wanting the same component on the x-axis in the grid, we need to have a map for each component, that way it lands in the same grid-column